gap/main.g

```diff
@@ -123,20 +121,26 @@ Chief := function( galoisField,mat,a,b,IsHPC )
         );
         TaskListPreClearUp := List(
             [ 1..b ],
-            x -> List( [ 1..b ], x -> dummyTask )
+            x -> List( [ 1..b ],
+                # simulates a task so that the entry can by used in `ScheduleTask`
```

Comment:
Further below the function `ScheduleTask` uses the entries of this matrix as conditions. A list of records is needed (tasks are special records). To guarantee this, we simulate completed tasks.

Reply:
See file `./hpcgap/lib/hpc/tasks.g` for the definition of tasks.

Usage of entries of TaskListPreClearUp:
```
327                             TaskListClearUpR[j][l][1] := ScheduleTask(                                                                                                                                                                    
328                                 [   TaskListPreClearUp[j][l],
329                                     TaskListPreClearUp[j][k]
330                                 ],
331                                 GAUSS_ClearUp_destructive,
332                                 R,
333                                 TaskResult( TaskListPreClearUp[j][k] ),
334                                 j,
335                                 k,
336                                 l
337                             );
```

Reply:
See file `./hpcgap/lib/hpc/tasks.g` for the definition of tasks.

Usage of entries of TaskListPreClearUp:
```
327                             TaskListClearUpR[j][l][1] := ScheduleTask(                                                                                                                                                                    
328                                 [   TaskListPreClearUp[j][l],
329                                     TaskListPreClearUp[j][k]
330                                 ],
331                                 GAUSS_ClearUp_destructive,
332                                 R,
333                                 TaskResult( TaskListPreClearUp[j][k] ),
334                                 j,
335                                 k,
336                                 l
337                             );
```